cpp/src/io/json/json_tree.cu

```diff
@@ -36,6 +36,7 @@
 #include <cub/device/device_radix_sort.cuh>
 
 #include <cuco/static_map.cuh>
+#include <cuco/static_set.cuh>
```

Comment:
Still need static map? If not, please remove the header above.

Reply:
replaced entired.